Automattic/wp-calypso
# Stats: Reduxify the stats post detail completely

Another Stats Route free of Stats List

In this PR, I update the stats post detail page to use Redux completely:
 - Reduxify StatsDetailMonths
 - Use `QueryPosts` instead of the stats response to load the post object

**Testing instructions**

 - Navigate to a page stats page `/stats/post/$postId/$site`
 - Check that the Title is showing up correctly
 - Check that the "Months & Years" and "Average by Day" panels are showing up properly
